src/models/meer7/internal-overview.md

```diff
@@ -0,0 +1,15 @@
+# Meerkat (Internal Overview)
+
+## Component overview:
+
+![Internal overview](./img/components.webp)
+
+- M.2 2280 (NVMe) SSD slot is highlighted in yellow
+- M.2 2242 (SATA) SSD slot is highlighted in cyan
+    - Used for the optional I/O add-on in the tall model
```

Comment:
Correct. The add-on uses the 2242 slot.

Reply:
Would saying "occupied by" instead of "used for" make it more clear?

Reply:
I thought it was plenty clear already, but that probably makes it even clearer

Reply:
I think the reason I didn't go with "occupied by" in the first place here was because that's only the case if it's installed, but I can just say "if installed."